device/celestica/x86_64-cel_silverstone-r0/Silverstone-128x100/pre-emphasis_PAM4_optics.soc

```diff
@@ -0,0 +1,2454 @@
+
+#***
+#*** Port ce0 Preemphasis setting ***
+#***
+
+local port ce0
+#*** lane 0 ***
+phy $port TXFIR_TAP_CTL0r.0 TXFIR_TAP0_COEFF=0

```

Comment:
using config.bcm?

Reply:
Broadcom FAE told me only the pre-emphasis of port was support in config,bcm for BH serdes, not for lane. But our SI team only gets the pre-emphasis of lane for BH serdes. so I have to use SOC file to handle it.
Thanks,
Bo